### DISCUSSION TITLE
Still no dotnet 8 support in Ionide?

### DISCUSSION THREAD

**Discussion by MarkuBu** [ionide/ionide-vscode-fsharp#2021] [General]
The Requirements still say

`.NET 6.0/7.0 SDK`

Because dotnet8 is LTS I want to use it. Is there any chance to get this up and running in VSCode?

**Comment by vzarytovskii** [comment]
I think it does support it. I use it to do some work on a project which is net8

**Comment by MarkuBu** [comment]
I get syntax highlighting and I can run code, but e.g. code formatting doesn't work.

I use Ubuntu 22.04 and I installed .NET via apt install dotnet8. I'm able to create projects and I can compile and run them. Bit inside VSCode everything becomes a bit weired.

Then I deinstalled dotnet8 (and all related packages) and installed dotnet7. Then I created a new project with dotnet7 and everything works as expected, in VSCode and even in Helix. I think, LSP doesn't work in dotnet8 as expected.

**Comment by MarkuBu** [comment]
I think I found the issue.  My version of fsautocomplete wasn't up to date. After updating to 0.73.2 also dotnet8 project seem to run as expected
